nav2_bringup/params/nav2_params.yaml

```diff
@@ -296,7 +298,8 @@ behavior_server:
       plugin: "nav2_behaviors/Wait"
     assisted_teleop:
       plugin: "nav2_behaviors/AssistedTeleop"
-    global_frame: odom
+    odom_frame: odom
```

Comment:
I'd prefer if these more 1-1 aligned with the global/local nomenclature established for their counterpart information. How about `local_reference_frame` and `global_reference_frame`?

Reply:
Or maybe just `local_frame` and `global_frame`?